content/academic_update/waiting/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: The Dynamics of Waiting
+math: true
+---
+
+In many daily situations, like e.g. in city centres or public transportation facilities, the dynamics of pedestrian crowds are dominated by the presence of standing or waiting pedestrians. However, pedestrian's waiting behaviour and interactions with moving persons has only rarely been studied. 
+For example in the fundamental diagram or in level of service concepts, waiting pedestrians are solely included while standing in queues. Standing in a queue with all surrounding pedestrians orienting in the same direction and probably sharing the same goal to reach, differs significantly from situations in which pedestrians can freely choose their waiting places, like for example at a train station platform.
+
+In order to increase the understanding of waiting pedestrians, we contributed a methodology and analysis to describe waiting passengers and their influence on the processes at railway platforms in the framework of the BMBF- founded Project "CroMa- Crowd Management in transportation infrastructures".  The detailed results can be found in publications  [[1]](#Küpper1),  [[2]](#Küpper2),  [[3]](#Küpper3), [[4]](#Küpper4) and will shortly be presented in the following sections.
+
+
+## Field studies
+Studying trajectory data from railway stations, which were kindly provided by Swiss Federal Railways (SBB AG), it became clear that the variability of types of users is important when describing how the space at railway platforms is used. While alighting passengers mostly leave the platform directly, boarding passengers spend a significant amount of time waiting. Therewith their trajectories cover a larger area of the platform and exhibit certain waiting places which can be identified by 'knots' in the trajectories.
+
+{{< figure src="traj_einaus.png" caption="Trajectories of boarding (red) and alighting passengers (blue) at Zürich Hardbrücke, Switzerland" >}}
```

Comment:
time period and number of people of visualized captured data would be nice to know

Reply:
Added time period.